templates/web/base/report/new/category_extras.html

```diff
@@ -1,5 +1,6 @@
 [% SET default_list = [] %][% FOR b IN bodies_to_list.values %][% default_list.push(b.cobrand_name) %][% END %]
 [% DEFAULT list_of_names = default_list %]
+[% category = mark_safe(category) %]
```

Comment:
Does this need to be earlier in the template process - e.g. the councils_text.html templates etc also look up `unresponsive.$category` which presumably could fail for the same reason.

Reply:
I've made sure they're all SafeStrings in d6a7f98c81a19baedc5305cb94483a052fd74b4f - does that look OK? Doesn't seem to matter if something is run through `mark_safe` twice, and makes it obvious in the template what's happening.

Reply:
It does double wrap, which it could perhaps not, but that doesn't have any effect on output, no.
Alternative, if you're always going to trust category anyway, would be to mark it safe in the perl code rather than in the templates? Meh, don't know if that'd be any easier/better.